test/unit/unit_smoke_test.py

```diff
@@ -1,4 +1,4 @@
-import exasol.developer_documentation
+import exasol.developer_documentation  # noqa: F401
```

Comment:
Does it make sense to add a comment why this is needed?
Otherwise readers need to lookup and search what F401 means....